src/migrations/20190412032715-create-ccc-change.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface
+            .createTable("CCCChanges", {
+                id: {
+                    allowNull: false,
+                    autoIncrement: true,
+                    primaryKey: true,
+                    type: Sequelize.BIGINT
+                },
+                address: {
+                    allowNull: false,
+                    type: Sequelize.STRING
+                },
+                change: {
+                    allowNull: false,
+                    type: Sequelize.NUMERIC({ precision: 20, scale: 0 })
+                },
+                blockNumber: {
+                    allowNull: false,
+                    type: Sequelize.INTEGER,
+                    onDelete: "CASCADE",
+                    references: {
+                        model: "Blocks",
+                        key: "number"
+                    }
+                },
+                reason: {
+                    allowNull: false,
+                    type: Sequelize.ENUM("fee", "author", "stake", "tx")
+                },
+                transactionHash: {
+                    allowNull: true,
+                    type: Sequelize.STRING,
+                    onDelete: "CASCADE",
+                    references: {
+                        model: "Transactions",
+                        key: "hash"
+                    }
+                },
+                createdAt: {
+                    allowNull: false,
+                    type: Sequelize.DATE
+                },
+                updatedAt: {
+                    allowNull: false,
+                    type: Sequelize.DATE
+                }
+            })
+            .then(() => {
+                return queryInterface.sequelize.query(
+                    'CREATE INDEX ccc_changes_block_number_idx on "CCCChanges"("blockNumber")'
+                );
+            });
```

Comment:
I think we should add an index for `transactionHash` for tx deletion performance.
And also for the `address`. Because we are going to query with an address.

BTW, is there a reason that you don't use `queryInterface.addIndex`?

Reply:
I added indexers for `transactionHash` and `address`.
I used `query` because the name `addIndex` generated is `c_c_c_...`, but I found an option to decide the indexer name and changed the code to use `addIndex`.